src/main/java/uk/gov/hmcts/reform/civil/handler/callback/camunda/notification/NotifyLIPDefendantClaimantSettleTheClaim.java

```diff
@@ -0,0 +1,74 @@
+package uk.gov.hmcts.reform.civil.handler.callback.camunda.notification;
+
+import lombok.RequiredArgsConstructor;
+import org.apache.commons.lang.StringUtils;
+import org.springframework.stereotype.Service;
+import uk.gov.hmcts.reform.ccd.client.model.AboutToStartOrSubmitCallbackResponse;
+import uk.gov.hmcts.reform.ccd.client.model.CallbackResponse;
+import uk.gov.hmcts.reform.civil.callback.Callback;
+import uk.gov.hmcts.reform.civil.callback.CallbackHandler;
+import uk.gov.hmcts.reform.civil.callback.CallbackParams;
+import uk.gov.hmcts.reform.civil.callback.CaseEvent;
+import uk.gov.hmcts.reform.civil.model.CaseData;
+import uk.gov.hmcts.reform.civil.notify.NotificationService;
+import uk.gov.hmcts.reform.civil.notify.NotificationsProperties;
+
+import java.util.Collections;
+import java.util.List;
+import java.util.Map;
+
+import static uk.gov.hmcts.reform.civil.callback.CallbackType.ABOUT_TO_SUBMIT;
+import static uk.gov.hmcts.reform.civil.callback.CaseEvent.NOTIFY_DEFENDANT_CLAIMANT_SETTLE_THE_CLAIM;
+import static uk.gov.hmcts.reform.civil.utils.PartyUtils.getPartyNameBasedOnType;
+
+@Service
+@RequiredArgsConstructor
+public class NotifyLIPDefendantClaimantSettleTheClaim extends CallbackHandler implements NotificationData {
+
+    private final NotificationService notificationService;
+    private final NotificationsProperties notificationsProperties;
+    private static final String REFERENCE_TEMPLATE = "notify-defendant-claimant-settle-the-claim-notification-%s";
+    public static final String TASK_ID = "NotifyDefendantClaimantSettleTheClaim";
+
+    @Override
+    protected Map<String, Callback> callbacks() {
+        return Map.of(
```

Comment:
we could move this to a final field so the map is not created every time we call the method. but created once with the bean. and return the field in this method

Reply:
made the changes